liveRecorder/scripts/build_ts2mp4_convertor.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [ "$#" -lt 2 ]; then
+	echo "usage build_ts2mp4_convertor <ffmpeg build path> <product path>"
+	exit
+fi
+
+FFMPEG_BUILD_PATH=$1
+PRODUCT_ROOT_PATH=$2
+TARGET=ts_to_mp4_convertor
+CONVERTOR_DIR=${PRODUCT_ROOT_PATH}/${TARGET}
+
+[ `uname` = "Linux" ] && OS="linux" || OS="darwin"
+
+export FFMPEG_BUILD_PATH
+export OS
+
+echo "OS=$OS"
+
+
+if [ -e $CONVERTOR_DIR ]; then
```

Comment:
If you want to check if a dir is readable, always use -r, for writeable, use -w.
-e is not enough and is almost never useful. For example:
```
jess@jessex:~$ whoami 
jess
jess@jessex:~$ ll -d /root/
drwx------ 67 root root 28672 Jun 22 13:08 /root//
jess@jessex:~$ if [ -e /root ];then echo exists;fi
exists
jess@jessex:~$ if [ -r /root ];then echo readable;fi
jess@jessex:~$ if [ -w /root ];then echo writeable;fi
```
So yes, /root exists but the jess user can't even read it so what good does it do that it exists?